secret_santa/api/views.py

```diff
@@ -1,3 +1,175 @@
-from django.shortcuts import render
+from django.http import Http404
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.views import APIView
 
-# Create your views here.
+from .models import Boxes, ReceiverSender, Users
+from .serializers import (BoxesSerializer, ReceiverSenderSerializer,
+                          UsersSerializer)
+
+
+class RoutesList(APIView):
+    """
+    List all routes.
+    """
+
+    def get(self, request, format=None):
+        routes = [
```

Comment:
You also have DELETE and POST methods to some of these endpoints

Reply:
Class RoutesList only for view endpoints, i don't think that he should have post, delete method. Correct me if I'm wrong